### PR TITLE
New Note added from #5469

### PR DESCRIPTION
**Pull request by j0rt3g4**
Issue #5469 
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mjcaparas** [comment]
I think it's better to state the required action rather than who can't access the functionality. 

Therefore, I suggest the following edit:

The option to upload a file to the library is only available to those with the appropriate RBAC permissions. The button is greyed out for users with only delegated permissions.

**Comment by j0rt3g4** [comment]
It's done @mjcaparas

**Comment by illfated** [comment]
Maybe I am mistaken, but does not this note belong either much further down, in the [**Put a file in the library**](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/live-response?source=docs#put-a-file-in-the-library) section (2nd half of the page), or at least after the bullet point "Ensure that you have the appropriate permissions" as exemplified below?
- **Ensure that you have the appropriate permissions**

File: windows/security/threat-protection/microsoft-defender-atp/live-response.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ You'll need to enable the live response capability in the [Advanced features set
     >[!WARNING]
     >Allowing the use of unsigned scripts may increase your exposure to threats.
  
+
+    > [!ÏMPORTANT]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/live-response.md]:
```suggestion
    > [!IMPORTANT]
```

**Comment by illfated** [comment]
@j0rt3g4 : Will you be able to edit and move the Note section from its current crowding position to below the "Ensure that you have the appropriate permissions" section? Or do you prefer to get suggestions from us to delete & add the new position?

**Comment by j0rt3g4** [comment]
@illfated  I haven't had the time yet but I will as soon as I can

**Comment by j0rt3g4** [comment]
@illfated  @JohanFreelancer9  @mypil done

**Comment by mypil** [comment]
@mjcaparas - Changes has been done. PR is now ready for final review. 

Thank you.

**Comment by mypil** [comment]
@mjcaparas - We would like to follow up. Can you please review and merge this PR? This is based on issue #5469

Thank you.

cc: @kenwith

**Comment by mjcaparas** [comment]
DOne

**Pull request merged**